scripts/configure_rancher_server.sh

```diff
@@ -2,6 +2,18 @@
 
 rancher_server_ip=${1:-172.22.101.100}
 orchestrator=${2:-cattle}
+cache_ip=172.22.101.101
+rancher_server_version=stable
+
+if [ ! "$(ps -ef | grep dockerd | grep -v grep | grep "$cache_ip")" ]; then
+  ros config set rancher.docker.registry_mirror "http://$cache_ip:5000"
+  ros config set rancher.system_docker.registry_mirror "http://$cache_ip:5000"
+  # bouncing the daemon results in indefinite crashloop, reboot instead
+  reboot now
```

Comment:
The only problem here is that provision needs to be executed over and over until the script completes. It needs to restart on reboot.